component/FloatingActionItem.js

```diff
@@ -13,12 +13,12 @@ import { getTouchableComponent } from './utils/touchable';
 class FloatingActionItem extends Component {
   constructor(props) {
     super(props);
-
+    console.log('props', props);
```

Comment:
let's remove this console.log :) ... I will be adding some prettier to prevent this error soon

Reply:
Oh I forgot about it, mb.

Great, thanks!